index.html

```diff
@@ -46,7 +51,19 @@ <h2>Sixteen <span>Clothing</span> </h2>
 
         </nav>
       </div>
+
+     
     </header>
+
+ 
+    <div class="scrolltoTop">
+      <button id="btnscrolltoTop">
+        
+        <span class="material-icons">
```

Comment:
replace this span with &lt;i class="fas fa-arrow-up-long"&gt;&lt;/i&gt;

Reply:
in google icons doesn't work by i element